packages/jest-util/src/fake_timers.js

```diff
@@ -229,7 +229,26 @@ export default class FakeTimers {
         'Ran ' +
           this._maxLoops +
           ' timers, and there are still more! ' +
-          "Assuming we've hit an infinite recursion and bailing out...",
+          "Assuming we've hit an infinite recursion and bailing out." +
+          'The following timers are still pending:' +
+          Object.keys(timers)
```

Comment:
`timers` is not defined, and the syntax here is weird.

Reply:
i know, im not sure if im doing this the right way, but i want to get the timers and then want to get only pending timers and output the stacktrace information for each specifically with map `const timers = Object.assign({}, this._timers);`